recipes/libcurl/all/conanfile.py

```diff
@@ -285,9 +304,53 @@ def _configure_autotools_vars(self):
 
             del autotools_vars['LIBS']
             self.output.info("Autotools env vars: " + repr(autotools_vars))
+
+        if tools.cross_building(self.settings):
+            if self.settings.os == "iOS": 
+                ios_dev_target = str(self.settings.os.version).split(".")[0]
+                if self.settings.arch in ["x86", "x86_64"]:
+                    iphoneos = "iphonesimulator"
+                    autotools_vars['CPPFLAGS'] = "-D__IPHONE_OS_VERSION_MIN_REQUIRED={}0000".format(ios_dev_target)
```

Comment:
we usually set something like `-mios-version-min`. there are helpers for that (as it's needed not just for iOS, but for watchOS/tvOS/macOS as well)

Reply:
found it, thanks, will update!

Reply:
using now `tools.apple_deployment_target_flag(self.settings.os, self.settings.os.version)` but still pass the define also, since this is proven to work